### ISSUE TITLE
Some minimal configurations broken since 6.8.1

### ISSUE THREAD

**Issue by james-criscuolo** [EddyVerbruggen/nativescript-plugin-firebase#932]
My `firebase.nativescript.json` is 
```
{
    "external_push_client_only": false,
    "using_ios": false,
    "using_android": true,
    "firestore": false,
    "realtimedb": false,
    "authentication": false,
    "remote_config": false,
    "messaging": true,
    "crashlytics": false,
    "crash_reporting": false,
    "storage": false,
    "functions": false,
    "facebook_auth": false,
    "google_auth": false,
    "admob": false,
    "invites": false,
    "dynamic_links": false,
    "ml_kit": false
}
```
When attempting to upgrade from 6.8.0 to latest (7.1.0), Upon init for android I get
```
TypeError: Cannot read property 'getInstance' of undefined
```

This started occurring between 6.8.0 and 6.8.1, and upon completely clean install (no platforms or node_modules directory). If I turn on authentication (seems to be the only change for 6.8.1), it seems like initialization works but then I get 
```
JS: APP ERROR HANDLER: Error: Uncaught (in promise): Uncomment firebase-messaging in the plugin's include.gradle first
JS: APP ERROR HANDLER: *** STACK TRACE BEGIN ***
JS: Error: Uncaught (in promise): Uncomment firebase-messaging in the plugin's include.gradle first
JS:     at resolvePromise (file:///data/data/<my-app>/files/app/tns_modules/nativescript-angular/zone-js/dist/zone-nativescript.js:825:31) [angular]
JS:     at resolvePromise (file:///data/data/<my-app>/files/app/tns_modules/nativescript-angular/zone-js/dist/zone-nativescript.js:782:17) [angular]
JS:     at file:///data/data/<my-app>/files/app/tns_modules/nativescript-angular/zone-js/dist/zone-nativescript.js:884:17 [angular]
JS:     at Object.onInvokeTask (file:///data/data/<my-app>/files/app/tns_modules/@angular/core/bundles/core.umd.js:3881:37) [angular]
JS:     at Function.drainMicroTaskQueue (file:///data/data/<my-app>/files/app/tns_modules/nativescript-angular/zone-js/dist/zone-nativescript.js:595:35) [<root>]
JS:     at Object.callback (file:///data/data/<my-app>/files/app/tns_modules/nativescript-angular/platform-common.js:143:25) [<root>]
JS:     at Observable.notify (file:///data/data/com.on...
JS: APP ERROR HANDLER: *** STACK TRACE END ***
```

i checked the include.gradle in the node_module, it is not commented out. I'm not sure what other information to provide, we compile android using SDK 26, but I tried bumping it to 27 and it caused the same issue. I also do not use sidekick, which had a recent fixed issue seemingly related to this.

**Comment by james-criscuolo** [comment]
I noticed that `init` is not called anywhere, but adding it allowed me to reproduce the error.

```
diff --git a/demo-push/app/push-view-model.ts b/demo-push/app/push-view-model.ts
index 2904cfc..ea0ac22 100644
--- a/demo-push/app/push-view-model.ts
+++ b/demo-push/app/push-view-model.ts
@@ -1,4 +1,5 @@
 import { Observable } from "tns-core-modules/data/observable";
+import { init } from "nativescript-plugin-firebase";
 import { messaging, Message } from "nativescript-plugin-firebase/messaging";
 import { alert, confirm } from "tns-core-modules/ui/dialogs";
 import * as platform from "tns-core-modules/platform";
@@ -26,6 +27,8 @@ export class PushViewModel extends Observable {
     if (applicationSettings.getBoolean(PushViewModel.APP_REGISTERED_FOR_NOTIFICATIONS, false)) {
       this.doRegisterPushHandlers();
     }
+
+    init();
   }
```

I guess it is unnecessary to call init if you only use messaging? If that is the case, I'll probably remove it from my project, but this used to work as of 6.8.0.

**Comment by james-criscuolo** [comment]
My issue appears to have been fixed with the release of 7.1.2, I'm now seeing the same issue as an existing issue:
https://github.com/EddyVerbruggen/nativescript-plugin-firebase/issues/928
So I will close this, and attempt to replicate over there. I believe the behavior also started with 6.8.1
